ui/src/components/DashboardAlerts.test.js

```diff
@@ -0,0 +1,94 @@
+import { STATUS_CRITICAL, STATUS_WARNING } from '../constants';
+import { render } from './__TEST__/util';
+import DashboardAlerts from './DashboardAlerts';
+import { fireEvent } from '@testing-library/react';
+
+const alertsCritical = [
+  {
+    id: 'alert1',
+    severity: STATUS_CRITICAL,
+  },
+];
+
+const alertsWarning = [
+  {
+    id: 'alert2',
+    severity: STATUS_WARNING,
+  },
+  {
+    id: 'alert3',
+    severity: STATUS_WARNING,
+  },
+];
+
+const mockOpenLink = jest.fn();
+
+jest.mock('../containers/AlertProvider', () => ({
+  __esModule: true,
+  default: ({ children }) => <>{children}</>,
+}));
+
+jest.mock('../containers/ConfigProvider', () => ({
+  __esModule: true,
+  default: ({ children }) => <>{children}</>,
+  useLinkOpener: () => ({
+    openLink: mockOpenLink,
+  }),
+  useDiscoveredViews: () => [
+    {
+      app: {
+        kind: '',
+        name: '',
+        version: '',
+        url: '',
+        appHistoryBasePath: '',
+      },
+      isFederated: true,
+      view: { path: '/alerts' },
+    },
+  ],
+}));
+
+describe('the dashboard alerts sub-panel', () => {
+  beforeEach(() => {
+    mockOpenLink.mockReset();
+  });
+
+  test('alerts should be displayed', () => {
```

Comment:
Ideally, we can form a sentence alone with the text in the `describe`
```suggestion
  test('should display the number of alerts, () => {
```
Same as the following scenarios.